pkg/cmd/server/origin/master.go

```diff
@@ -656,6 +656,7 @@ func (c *MasterConfig) GetRestStorage() map[string]rest.Storage {
 	buildConfigWebHooks := buildconfigregistry.NewWebHookREST(
 		buildConfigRegistry,
 		buildclient.NewOSClientBuildConfigInstantiatorClient(bcClient),
+		v1.SchemeGroupVersion,
```

Comment:
This should point explictly to the external group version from `pkg/build/api/v1`.